src/main/java/de/unijena/cheminf/mortar/gui/views/SettingsView.java

```diff
@@ -312,6 +314,18 @@ protected void updateItem(IDisplayEnum item, boolean empty) {
                 default ->
                         throw new UnsupportedOperationException("Unknown property type " + tmpProperty.getName());
             }
+
+            Region backgroundRegion = new Region();
+            backgroundRegion.setPrefHeight(40);
+
+            // Set alternating colors
+            if (tmpRowIndex % 2 == 0) {
+                backgroundRegion.setStyle("-fx-background-color: #f5f5f5;");
+            } else {
+                backgroundRegion.setStyle("-fx-background-color: #ffffff;");
```

Comment:
I would prefer to start with white, so that the head of the grid looks more defined

Reply:
(this is how I realised that the first row is ignored right now in the styling, see comment below).